redaxo/src/core/lib/minibar/minibar.php

```diff
@@ -70,8 +69,11 @@ public static function isVisible()
      */
     public static function setVisibility($value)
     {
-        $minibar = ['visible' => $value];
-        rex::getProperty('login')->setSessionVar('minibar', $minibar);
+        if ($value) {
+            rex_response::sendCookie('rex_minibar_visibility', '1', ['expires' => rex::getProperty('session_duration'), 'samesite' => 'strict']);
```

Comment:
@staabm @gharlan 
Habt ihr eine Idee warum das Cookie nicht gesetzt wird.

Reply:
Lösung gefunden in https://github.com/redaxo/redaxo/pull/1921/commits/85105727953364ab7ef06930443c5c4bd2c7bee2